core/internal/testutils/pgtest/pgtest.go

```diff
@@ -29,9 +30,24 @@ func NewSqlDB(t *testing.T) *sql.DB {
 	return db
 }
 
-func NewSqlxDB(t testing.TB) *sqlx.DB {
+func NewEVMScopedDB(t testing.TB) *sqlx.DB {
+	// hack to scope to evm schema. the value "evm" will need to be dynamic to support multiple relayers
+	url := pg.SchemaScopedConnection(defaultDBURL, "evm")
+	return NewSqlxDB(t, WithURL(url))
+}
+
+func NewSqlxDB(t testing.TB, opts ...ConnectionOpt) *sqlx.DB {
 	testutils.SkipShortDB(t)
-	db, err := sqlx.Open(string(dialects.TransactionWrappedPostgres), uuid.New().String())
+	conn := &pg.ConnectionScope{
+		UUID: uuid.New().String(),
+		URL:  defaultDBURL,
+	}
+	for _, opt := range opts {
+		opt(conn)
+	}
+	enc, err := json.Marshal(conn)
+	require.NoError(t, err)
+	db, err := sqlx.Open(string(dialects.TransactionWrappedPostgres), string(enc))
```

Comment:
Not sure I understand this... Why can't we just modify the connection string and add the schema to it here, rather than passing in a json blob?